gatorgrade/output/output_tools.py

```diff
@@ -1,10 +1,9 @@
-"""Add docstring."""
+"""
+This file is used for storing supporting functions and tools.
 
-
-# This file is used for storing supporting functions
-# and tools that are used by the methods in output_functions.
-# The main use of this class is to de-clutter
-# output_functions.py for better readability.
-# One example of a function to be written here is the
-# splitting of strings retrieved from the input team,
-# at spaces, into lists.
+These functions and tools are used by the methods in output_functions.
+The main use of this class is to de-clutter output_functions.py for
+better readability. One example of a function to be written here is
+the splitting of strings retrieved from the input team, at spaces, into
+lists.
+"""

```

Comment:
Can this follow google docstring styling?

Reply:
Could you elaborate on this point?  Do you mean changing the docstring to look like this?  Or as something else?

```
"""This file is used for storing supporting functions and tools.

# This file is used for storing supporting functions
# and tools that are used by the methods in output_functions.
# The main use of this class is to de-clutter
# output_functions.py for better readability.
# One example of a function to be written here is the
# splitting of strings retrieved from the input team,
# at spaces, into lists.
These functions and tools are used by the methods in output_functions.
The main use of this class is to de-clutter output_functions.py for
better readability. One example of a function to be written here is
the splitting of strings retrieved from the input team, at spaces, into
lists.
"""

```

Reply:
Hello @ullrichd21! Could you please clarify whether @gvanzin-allegheny's modification addresses your feedback?

Reply:
> 

This looks perfect, thanks!